packages/rest/test/integration/http-handler.integration.ts

```diff
@@ -521,26 +527,11 @@ describe('HttpHandler', () => {
   function givenClient() {
     client = createClientForHandler((req, res) => {
       handler.handleRequest(req, res).catch(err => {
-        console.error('Request failed.', err.stack);
+        debug('Request failed.', err.stack);
```

Comment:
This console log must be preserved, see my comment above. If there is a test that's triggering this code path as part of the expected operation, then please use a mechanism similar to `skipStatusCode` above to allow tests to control which errors to silently ignore.